gulpfile.js

```diff
@@ -150,37 +147,34 @@ gulp.task('lint-scss', () => {
  *
  */
 gulp.task('import-changelog-file', () => {
-  let url = "https://api.github.com/repos/youseedk/dna/releases";
-  if(githubClientId && githubClientSecret) {
+  let url = 'https://api.github.com/repos/youseedk/dna/releases';
+  if (githubClientId && githubClientSecret) {
     url = `${url}?client_id=${githubClientId}&client_secret=${githubClientSecret}`;
   }
-  return request(
-    {
-      url: url,
-      method: "get",
-      headers: {
-        "user-agent": "User-Agent"
-      }
-    },
-    function(error, response, body) {
-      if(error) {
-        console.error(error);
-      }
-    }
-  )
-  .pipe(source('github-releases.json'))
-  .pipe(gulp.dest(`${paths.tokensSource}/generated`));
-})
-
+  return request({
+        url: url,
+        method: 'get',
+        headers: {
+          'user-agent': 'User-Agent'
+        }
+      },
+      function (error) {
+        if (error) {
+          console.error(error);
+        }
+      })
+    .pipe(source('github-releases.json'))
+    .pipe(gulp.dest(`${paths.tokensSource}/generated`));
+});
 
 /**
-
+ *
  *	Convert colors.json file to .scss-file
  *
  */
 gulp.task('jsonToScss', () => {
   return gulp.
-    src([`${paths.tokensSource}colors.json`])
+  src([`${paths.tokensSource}colors.json`])

```

Comment:
This needs indentation and was correct?

Reply:
No this is just a linebreak and shold not be indented

Reply:
It's indented everywhere else in the file...

Reply:
It should be indented if we move the dot to this line ;)